src/RA_Dlg_MemBookmark.cpp

```diff
@@ -539,31 +536,38 @@ void Dlg_MemBookmark::SetupColumns(HWND hList)
 
     //	Remove all data.
     ListView_DeleteAllItems(hList);
+    {
+        auto lplvColumn{ std::make_unique<LV_COLUMN>() };
```

Comment:
Why move this to the heap?

Reply:
It used ZeroMemory before so I thought it needed an extendend lifetime. From now on everything will be stack based based. Using constructors instead of ZeroMemory.